com.super_productivity.SuperProductivity.metainfo.xml

```diff
@@ -2,7 +2,7 @@
 <component type="desktop">
   <id>com.super_productivity.SuperProductivity</id>
   <name>Super Productivity</name>
-  <summary>Advanced to-do list</summary>
+  <summary>To-do list & time tracker</summary>
```

Comment:
Haven't run the checker locally to see if this is the problem, but trying it anyway:
```suggestion
  <summary>To-do list and time tracker</summary>
```